Earthfile

```diff
@@ -110,6 +110,24 @@ docker:
     SAVE IMAGE --push $CI_REGISTRY_IMAGE:$GIT_COMMIT
     SAVE IMAGE --push $CI_REGISTRY_IMAGE:$GIT_TAG
 
+dlv:
+    FROM golang:1.19
```

Comment:
want to pass the version in from .tool-versions, for consistency?

Reply:
good idea